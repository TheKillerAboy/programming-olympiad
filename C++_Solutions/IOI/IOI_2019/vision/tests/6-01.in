c2675211-ade0-44b0-8c15-741dd835f3d2
2 2 1
3 128
128 129
128 130
128 131
