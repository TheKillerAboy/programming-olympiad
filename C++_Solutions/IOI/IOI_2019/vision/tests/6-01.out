b17553fd-ba5a-4140-836c-491f938c515b
OK
60
1 1 0
2 1 0
1 2 1 2
2 2 1 2
1 1 3
2 1 3
1 1 1
2 1 1
1 2 0 3
2 2 0 3
1 1 2
2 1 2
2 1 11
1 1 10
3 16
0 2 17 18
2 1 13
1 1 12
3 20
0 2 21 22
2 1 15
1 1 14
3 24
0 2 25 26
1 3 19 23 27
2 1 5
1 1 4
3 29
0 2 30 31
2 1 7
1 1 6
3 33
0 2 34 35
2 1 9
1 1 8
3 37
0 2 38 39
1 3 32 36 40
0 2 41 28
2 2 11 13
1 2 10 12
3 43
0 2 44 45
2 2 15 13
1 2 14 12
3 47
0 2 48 49
1 2 46 50
2 2 5 7
1 2 4 6
3 52
0 2 53 54
2 2 9 7
1 2 8 6
3 56
0 2 57 58
1 2 55 59
0 2 60 51
3 42
0 2 61 62
